lib/app/utils/validation.rb

```diff
@@ -75,7 +75,8 @@ def self.definitions
       head_circumference: 'http://hl7.org/fhir/StructureDefinition/headcircum',
       body_weight: 'http://hl7.org/fhir/StructureDefinition/bodyweight',
       bmi: 'http://hl7.org/fhir/StructureDefinition/bmi',
-      blood_pressure: 'http://hl7.org/fhir/StructureDefinition/bp'
+      blood_pressure: 'http://hl7.org/fhir/StructureDefinition/bp',
+      vital_signs: 'http://hl7.org/fhir/StructureDefinition/vitalsigns'
```

Comment:
I thought we had decided that US Core and USCDI does not require the 'generic' vital signs profile, so we do not have any regulatory or standards backing to fail them if they send a resource that looks like a vital sign by having a vital-signs coding (but do not fall into one of the profiles that are required).  Or did we find something that would give us that ability (I really doubt it because g10 is very specific about only requiring certain profiles that USCDI maps to, and vital signs isn't one of them).

Reply:
Yes. This is a valid concern. So another way to address this is adding additional logic in bulk data sequence.